tests/conftest.py

```diff
@@ -109,7 +108,7 @@ def manifest_with_matching_deployment(w3, tmpdir, safe_math_manifest):
             "block": "0x420cb2b2bd634ef42f9082e1ee87a8d4aeeaf506ea5cdeddaa8ff7cbf911810c",
         }
     }
-    return manifest
+    return manifest, w3
```

Comment:
Why return `w3` also?  Seems like you could just use the `w3` fixture wherever you use this fixture to get the `w3` instance?  Or are there multiple `w3` instances floating around?